msteams-platform/apps-in-teams-meetings/API-references.md

```diff
@@ -302,6 +305,185 @@ The JSON response body for Meeting Details API is as follows:
+
+# [C#](#tab/dotnet)
+
+Not available
+
+# [JavaScript](#tab/javascript)
+
+```typescript
+
+const appContentUrl = "https://www.bing.com/";
+
+microsoftTeams.meeting.shareAppContentToStage ((err, result) => {
+if(result) {
+  this.setState({ isAppSharing: true });
+ }
+if(err) {
+  this.setState({ sharingError: err, isAppSharing: false })
+ }
+}, appContentUrl); 
+
+```
+
+# [JSON](#tab/json)
+
+Not available
+
+---
+
+### Response codes
+
+The `shareAppContentToStage` API returns the following response codes:
+
+|Response code|Description|
+|---|---|
+| **500** | Internal error. |
+| **501** | API is not supported in the current context.|
+| **1000** | App does not have proper permissions to allow share to stage.|
+
+## getAppContentStageSharingState API
+
+The `getAppContentStageSharingState` API enables you to fetch information about apps' sharing on the meeting stage.
+
+### Query parameter
+
+The `getAppContentStageSharingState` API includes the following parameter:
+
+|Value|Type|Required|Description|
+|---|---|----|---|
+|**callback**| String | Yes | Callback contains two parameters, error and result. The *error* can either contain an error of type *SdkError*, in case of an error, or null when share is successful. The `appContentStageSharingState` can either contain an `AppContentStageSharingState` object, indicating successful retrieval, or null, indicating failed retrieval.|
+
+### Example
+
+The following codes provide examples of `getAppContentStageSharingState` API:
+
+# [C#](#tab/dotnet)
+
+Not available
+
+# [JavaScript](#tab/javascript)
+
+```microsoftTeams.meeting.getAppContentStageSharingState((err, result)) => {
+  if(result.isAppSharing) {
+    this.setState({ isGameSessionOver: false });
+   }
+  });
+``` 
+
+# [JSON](#tab/json)
+
+Not available
+
+---
+
+The JSON response body for the `appContentStageSharingState` API is:
+
+```json
+{
+   "isAppSharing":true
+  }
+  
+```
+
+
+### Response codes
+
+The `getAppContentStageSharingState` API returns the following response codes:
+
+|Response code|Description|
+|---|---|
+| **500** | Internal error. |
+| **501** | API is not supported in the current context.|
+| **1000** | App does not have proper permissions to allow share to stage.|
+
+## getAppContentStageSharingCapabilities API
+
+The `getAppContentStageSharingCapabilities` API enables you to fetch the apps' capabilities for sharing to meeting stage.
+
+### Query parameter
+
+The `getAppContentStageSharingCapabilities` includes the following parameters:
+
+|Value|Type|Required|Description|
+|---|---|----|---|
+|**callback**| String | Yes | Callback contains two parameters, error and result. The *error* can either contain an error of type *SdkError*, in case of an error, or null when share is successful. The `appContentStageSharingState` can either contain an `AppContentStageSharingState` object, indicating successful retrieval, or null, indicating failed retrieval.|
```

Comment:
@kdugue-msft -- Is this accurate for getAppContentSharingCapabilities callback? 
_The `appContentStageSharingState` can either contain an `AppContentStageSharingState` object, indicating successful retrieval, or null, indicating failed retrieval._

Reply:
Same as the previous comment, since we use `error` and `result`, we can stay consistent here.

Suggestion:
The result can either contain an AppContentStageSharingState object, indicating successful retrieval, or null, indicating failed retrieval.